app/models/financial_aid/my_awards.rb

```diff
@@ -233,7 +248,12 @@ def alert_details(item_type)
     end
 
     def disbursements(item_type)
-      disbursements ||= EdoOracle::FinancialAid::Queries.get_awards_disbursements(@uid, my_aid_year, item_type)
+      if item_type
+        disbursements ||= EdoOracle::FinancialAid::Queries.get_awards_disbursements(@uid, my_aid_year, item_type)
```

Comment:
@marcpechaitis - Same thing here. Are you trying to cache these within the instance, because they're called over and over again?